.github/workflows/linux.yml

```diff
@@ -430,6 +421,14 @@ jobs:
 
     steps:
 
+      - name: Fetch setup_python action
```

Comment:
imho such a step should be connected to the action step (put them near each other)

Reply:
ideally we need to make this action public and do not check out it every time